File: Python/1051.py
salary = float(input())

if 0.00 <= salary <= 2000.00:
    print("Isento")
elif 2000.01 <= salary <= 3000.00:
    print("R$ {:.2f}".format((salary - 2000) * 0.08))
elif 3000.01 <= salary <= 4500.00:
    print("R$ {:.2f}".format((salary - 3000) * 0.18 + (1000 * 0.08)))
elif salary > 4500.00:
    print("R$ {:.2f}".format((salary - 4500) * 0.28 + (1500 * 0.18) + (1000 * 0.08)))

File: Dart/1051.dart
import 'dart:io';

void main() {
  double salary = double.parse(stdin.readLineSync());

  if (salary <= 2000) {
    print("Isento");
  } else if (salary > 2000 && salary <= 3000) {
    print("R\$ ${((salary - 2000) * 0.08).toStringAsFixed(2)}");
  } else if (salary > 3000 && salary <= 4500) {
    print("R\$ ${((salary - 3000) * 0.18 + (1000 * 0.08)).toStringAsFixed(2)}");
  } else if (salary > 4500) {
    print(
        "R\$ ${((salary - 4500) * 0.28 + (1500 * 0.18) + (1000 * 0.08)).toStringAsFixed(2)}");
  }
}
